ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nightly-4.13-loaded-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,57 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-ld
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  initial:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.12.0-0
+        upper: 4.13.0-0
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-loaded-upgrade-aws
+  cron: 0 12 * * 1
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      CLEANUP_WHEN_FINISH: "false"
+      ENABLE_FORCE: "true"
+      MAX_UNAVAILABLE: "3"
+      WORKER_REPLICA_COUNT: "5"
```

Comment:
Our normal loaded-upgrade tests run with 50 workers. Think this is good for now to get the functionality in but will want to update as we gain confidence in the scaling of workers

Reply:
Ok, Let's keep the job running for some while, then update to 50 later, thanks!

Reply:
Let's keep the job running for some while, then upgrade to 50 worker nodes later, should we replace all current jenkins loaded upgrade job later?

Reply:
Yes! but since we are adding the functionality here we can add scaling up to larger numbers later